tests/testthat/_snaps/tidy-upkeep.md

```diff
@@ -69,3 +69,51 @@
       * [ ] Add alt-text to pictures, plots, etc; see https://posit.co/blog/knitr-fig-alt/ for examples
       
 
+# upkeep bullets don't change accidentally
+
+    Code
+      writeLines(upkeep_checklist())
+    Output
+      
+      * [ ] `usethis::use_readme_rmd()`
+      * [ ] `usethis::use_github_links()`
+      * [ ] `usethis::use_pkgdown_github_pages()`
+      * [ ] `urlchecker::url_check()`
+      * [ ] `usethis::use_package_doc()`
+      Consider letting usethis manage your `@importFrom` directives here.
+      `usethis::use_import_from()` is handy for this.
```

Comment:
Does this mean it's going to look wonky in the GitHub issue too?

Reply:
Here's an [example issue](https://github.com/ateucher/rmapshaper/issues/144). I used the same formatting of the those bullets as in the original `tidy_upkeep_checklist()` - some have line breaks, and some have all on one line...

Reply:
That all looks fine to me.